### PR TITLE
Master

### PR DESCRIPTION
**Pull request by ytechie**
Updated the 201 level templates (and tested) to include a link to the Azure Resource Visualizer.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ytechie__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: 201-2-vms-internal-load-balancer/README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-internal-load-balancer%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-internal-load-balancer%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-2-vms-internal-load-balancer/README.md]: This one doesn't look right. Doesn't show the VMs under a load balancer. I suggest removing the visualize button for now.

File: 201-2-vms-loadbalancer-lbrules/README.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-loadbalancer-lbrules%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-loadbalancer-lbrules%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-2-vms-loadbalancer-lbrules/README.md]: Same as above, let's leave this for now

File: 201-2-vms-loadbalancer-natrules/README.md [hunk 3]
```diff
@@ -3,6 +3,9 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-loadbalancer-natrules%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-2-vms-loadbalancer-natrules%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-2-vms-loadbalancer-natrules/README.md]: Same as above

File: 201-nsg-dmz-in-vnet/Readme.md [hunk 4]
```diff
@@ -2,4 +2,7 @@ This template creates a VNet with 3 Subnets, Frontend, App Subnet and Database s
 
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-nsg-dmz-in-vnet%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-nsg-dmz-in-vnet%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-nsg-dmz-in-vnet/Readme.md]: I noticed VNET is called VPN on the tool. Is there a reason? Seems to cause confusion when it's called VNET in JSON and VPN in visual

File: 201-rbac-builtinrole-multipleVMs/ReadMe.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-rbac-builtinrole-multipleVMs%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-rbac-builtinrole-multipleVMs%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-rbac-builtinrole-multipleVMs/ReadMe.md]: Doesn't render

File: 201-site-to-site-vpn/README.md [hunk 6]
```diff
@@ -3,6 +3,9 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-site-to-site-vpn%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-site-to-site-vpn%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-site-to-site-vpn/README.md]: Empty boxes rendered, needs additional work

File: 201-vm-different-rg-vnet/README.md [hunk 7]
```diff
@@ -3,5 +3,8 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vm-different-rg-vnet%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vm-different-rg-vnet%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-vm-different-rg-vnet/README.md]: This should be a feature request for the tool to show that the VNET exists outside current RG boundary. Remove for now.

File: 201-vm-domain-join/Readme.md [hunk 8]
```diff
@@ -12,3 +12,6 @@ Click the button below to deploy
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vm-domain-join%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vm-domain-join%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-vm-domain-join/Readme.md]: Doesn't render correctly.

File: 201-windows-vm-diagnostics-extension/README.md [hunk 9]
```diff
@@ -3,5 +3,8 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-premium-storage-windows-vm-diagnostics-extension%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-windows-vm-diagnostics-extension/azuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-windows-vm-diagnostics-extension/README.md]: Shows a storage account that is not connected to the VM. Can you check what's going on?

**Comment by ytechie** [comment]
Thank you for the feedback. I'm going to close this for now. I'm adding unit tests and will resend later.

**Pull request closed without merging**